modules/javafx.graphics/src/main/native-glass/mac/GlassViewDelegate.m

```diff
@@ -1046,26 +1046,26 @@ - (void)startDrag:(NSDragOperation)operation withItems:(NSArray<NSDraggingItem*>
 
         if (image != nil)
         {
-            // select the center of the image as the drag origin
```

Comment:
There are two consecutive `if (image != nil)` blocks, so you can remove the 4 lines above this one.